src/sign.ml

```diff
@@ -254,6 +254,16 @@ let read : string -> t = fun fname ->
     its dependencies. *)
 let add_rule : t -> sym -> rule -> unit = fun sign sym r ->
   sym.sym_rules := !(sym.sym_rules) @ [r];
+  (* EXPE *)
+  if List.length !(sym.sym_rules) > 0 then
```

Comment:
This condition is always true. A rule was just added on line 256.

Reply:
Indeed, it has been corrected in 20a650a.